tirocks-sys/src/lib.rs

```diff
@@ -10,6 +10,10 @@ extern crate bzip2_sys;
 
 #[allow(clippy::all)]
 mod bindings {
+    pub enum rocksdb_TableProperties {}
```

Comment:
why?

Reply:
It's explained in #6 that these bindings are blocklisted and opaque, so they will not be generated by rust-bindgen.

Reply:
Miss it. I think we should generate bindings for the different platforms instead of this workaround.

Reply:
It already does. It just reuses the same bindings for Linux and MacOS. Beside, generating these bindings will not make things simpler as we need to take care of various implicit constructors and destructors.